### DISCUSSION TITLE
Welcome to hamonize Discussions!

### DISCUSSION THREAD

**Discussion by gon1942** [hamonikr/hamonize#530] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

우리는 토론을 커뮤니티의 다른 구성원과 연결하는 장소로 사용하고 있습니다. 우리는 당신이:

궁금한 점을 물어보세요.
아이디어를 공유하세요.
다른 커뮤니티 구성원과 교류하십시오.
다른 사람을 환영하고 개방적입니다. 이 커뮤니티는 우리가
함께 구축 .
시작하려면 아래에 자신에 대한 소개와 함께 댓글을 달고 이 커뮤니티에서 하는 일에 대해 알려주세요.

**Comment by loyal2linux** [comment]
How do I change the interface from Korean to English?  My wife is Korean and my Korean skills  are weak.  So I need to set up Harmoni in both languages.  I cant find my way through the korean interface to do this.  Can someone help?

If I can do this I will talk to our school to set up our linux machines with Harmoni as we have Korean students.  Also would like to help some churches who use windows illegally to change to a korean version of Linux.  

is there software to recreate the iso after customizing it.  I am a MX-linux user. 

Thanks Kamsahamneedah.  
losterhoudt@crics.asia
